showcase/korogenoest.md

```diff
@@ -0,0 +1,130 @@
+
+## Data brokering to ENA 
+
+To upload data to [European Nucleotide Archive (ENA)](https://www.ebi.ac.uk/ena/browser/) we registered our research [projects at the ENA](https://www.ebi.ac.uk/ena/browser/text-search?query=korogeno-est) following the [guideline](https://ena-docs.readthedocs.io/en/latest/submit/study.html). At the beginning we created individual projects for a data batch but later joined many under one. The Data broker generated ENA metadata files using ENA virus pathogen reporting standard checklist ERC000033 filling out mandatory and recommended fields ([checklist](https://www.ebi.ac.uk/ena/browser/view/ERC000033), [metadata template](https://github.com/ELIXIR-Belgium/ENA-metadata-templates)).  The preparatory work for data brokering to ENA was done by a dedicated Galaxy workflow. A typical workflow of extraction of Galaxy results to the metadata file and metadata ERC000033 file generation took about 1h. 
+
+Out of the three different raw sequencing data uploading ways, Estonia used the Galaxy-CLI instance which is a wrapper for the ENA upload tool. The wrapper is [available](https://github.com/usegalaxy-eu/ena-upload-cli) along with the [dedicated guide](https://rdm.elixir-belgium.org/covid-19/sarscov2_submission.html) that was [published](https://doi.org/10.1093/bioinformatics/btab421) in 2021.
+
+
+{% include image.html file="/recommended_dataflow.png" caption="An overview of SARS-CoV-2 raw reads and metadata upload steps (a-d) to ENA enabled by Galaxy. Adapted from [A SARS-CoV-2 sequence submission tool for the European Nucleotide Archive](https://doi.org/10.1093/bioinformatics/btab421) publication" alt="Galaxy ENA uploader takes raw reads as fastq files and metadata spreadsheet as an input. Then human reads are removed from the sequence data and the raw sequences gets uploaded to ENA along with the accompanying metadata. After variation analysis and consensus sequence workflow execution also consensus sequences get uploaded to ENA.
+" %}
+
+
+All the sequenced raw reads results were uploaded to the University of Tartu Galaxy instance. From there the data manager selected a batch and successfully sequenced samples into a working collection. Forward and reverse sequences were paired up and run through the workflow of dehumanisation of raw sequence reads (uses Metagen-FastQC). 
+
+In Galaxy ENA uploader metadata checklist (ERC000033) was marked and the filled-out template. The runs input format was selected from a paired collection (that was human sequence free), an affiliation center was added, and the ENA upload tool was run.
```

Comment:
```suggestion
The ERC000033 metadata checklist was used and the corresponding template was filled. The runs input format was selected from a paired collection (that was human sequence free), an affiliation center was added, and the ENA upload tool was run.
```

Reply:
In the ENA uploader you cannot fill any template. You just upload pre-filled file.
So we marked from the dropdown list that it belonged to the ERC000033 checklist and uploaded our file.